src/init.cpp

```diff
@@ -853,12 +853,15 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         nLocalServices = ServiceFlags(nLocalServices | NODE_COMPACT_FILTERS);
     }
 
-    // if using block pruning, then disallow txindex and coinstatsindex
+    // if using block pruning, then disallow txindex and coinstatsindex and reindex-chainstate
```

Comment:
Actually I think it could make sense to remove it here. Looks like #24539 forgot to update the comment, so one option would be to go through the pain of updating #24539 (and causing more merge conflicts), or the other option to just remove it? wdyt?

Reply:
Makes sense, I removed it - I agree that it is self-explanatory and only leads to merge conflicts.